internal/js/modules/k6/browser/common/element_handle_test.go

```diff
@@ -157,33 +156,42 @@ func TestQueryAll(t *testing.T) {
 	t.Run("return_in_document_order", func(t *testing.T) {
 		t.Parallel()
 
-		const numElements = 15
-
+		const numElements = 11
 		elems := make([]*ElementHandle, numElements)
-		handles := map[string]JSHandleAPI{
-			// non-numeric keys that should be filtered out
-			"length":  &jsHandleStub{asElementFn: nilHandle},
-			"forEach": &jsHandleStub{asElementFn: nilHandle},
-			"item":    &jsHandleStub{asElementFn: nilHandle},
-		}
 		for i := range numElements {
 			elems[i] = &ElementHandle{}
-			key := strconv.FormatInt(int64(i), 10)
-			handles[key] = &jsHandleStub{
-				asElementFn: func(idx int) func() *ElementHandle {
-					return func() *ElementHandle { return elems[idx] }
-				}(i),
+		}
+		asElementFn := func(elem *ElementHandle) func() *ElementHandle {
+			{
+				return func() *ElementHandle { return elem }
 			}
 		}
 
-		returnHandle := &jsHandleStub{
-			getPropertiesFn: func() (map[string]JSHandleAPI, error) {
-				return handles, nil
-			},
+		handles := map[string]JSHandleAPI{
+			// numeric keys
+			"0":  &jsHandleStub{asElementFn: asElementFn(elems[0])},
```

Comment:
I like the way these look as a flat list, so we don't need to think about the input 👍